crates/turbopack-ecmascript/src/analyzer/mod.rs

```diff
@@ -2869,6 +2830,7 @@ pub enum WellKnownObjectKind {
     NodePreGyp,
     NodeExpressApp,
     NodeProtobufLoader,
+    NodeBuffer,
```

Comment:
is this just for testing?

Reply:
Its also used here: https://github.com/vercel/turbo/blob/07bf57235cdda2cc973e35b646464ee1847d1403/crates/turbopack-ecmascript/src/references/mod.rs#L1832

but there is no special behavior for it defined yet. But it could static evaluate e. g. `Buffer.from("SGVsbG8gV29ybGQ=", "base64").toString("utf-8")` in future, but I don't see a need for that right now.